Microsoft/edx-configuration
# Master upstream sync

#### What's this PR do? Any additional context?

Syncing our master branch with edx to cherry pick some fixes for continuous integration AND tests. This also helps us towards upstream contributions

#### Where should the reviewer start?

Changes in .travis file. The rest can be safely ignored (or cursory reviewed) for now because most of it will not make its way into our oxa/*fic branches for a while.

#### How can this be manually tested? (brief repro steps)

This is the first of two changes that will fix travis test runs. Therefore, you can verify this works when you submit a pull requests to our repo sometime next week.

#### What are the relevant TFS items? (list id numbers)

99887

#### Definition of done:

[//]: # ( todo: Is there appropriate test coverage? )
[//]: # ( todo: Does this PR require a new Selenium test? )
[//]: # ( todo: Is there appropriate logging/monitoring included? )

#### Reminders BEFORE merging
1. Get at least two approvals
1. If you're merging into the development branch then "flatten" or "squash" commits
1. If merging from development into master then don't "flatten" or "squash" commits

#### Reminders AFTER merging
1. Delete the remote branch
1. Resolve relevant TFS items
1. (reverse merge) If you merged into master then check to see if there are any changes in master that can be merged down to the development branch (like hotfixes, etc)

[//]: # ( todo: If you merged into development branch then verify change in our "rolling deployment" environment. Then notify stakeholders interested in or involved with the change )


[//]: # ( fyi: This content was heavily inspired by )
[//]: # ( 1 Our team's policies and processes )
[//]: # ( 3 https://github.com/sprintly/sprint.ly-culture/blob/master/pr-template.md )
[//]: # ( 4 The book "The Checklist Manifesto: How to Get Things Right" by Atul Gawande )
[//]: # ( 5 https://github.com/Azure/azure-event-hubs/blob/master/.github/PULL_REQUEST_TEMPLATE.md )


Configuration Pull Request
---
#### (For changes proposed to upstream)
Make sure that the following steps are done before merging
